main.tf

```
@@ -48,7 +48,6 @@ resource "google_compute_region_backend_service" "default" {
   name                            = var.health_check["type"] == "tcp" ? "${var.name}-with-tcp-hc" : "${var.name}-with-http-hc"
   region                          = var.region
   protocol                        = var.ip_protocol
```

Let's replace this with a comment so people don't accidentally re-add it.

```suggestion
  protocol                        = var.ip_protocol
  # Do not try to add timeout_sec, as it is has no impact. See https://github.com/terraform-google-modules/terraform-google-lb-internal/issues/53#issuecomment-893427675S
```